lib/pluginOverride.js

```diff
@@ -66,7 +66,7 @@ module.exports = function override (old, fn, opts) {
     instance[kFourOhFour].arrange404(instance)
   }
 
-  for (const hook of instance[kHooks].onRegister) hook.call(this, instance, opts)
+  for (const hook of instance[kHooks].onRegister) hook.call(old, instance, opts)
```

Comment:
This looks breaking to me.

Reply:
Yes, what I meant is that since the final versions of the core plugins were released, I made sure to run `citgm-core-plugins` and they all passed, so nothing _frozen_ appears to have been broken by this change

And maybe I'm not knowledgeable enough here, but taking a step back, I personally haven't seen `this` used with onRegister ever, and I think it's not just because it was undocumented — passing the avvio instance there doesn't seem useful for a user

I suspect the goal was to pass the parent in the original implementation; if so, we could even call this a bug fix

But anyway, I simply think this is a good consistency improvement that we can fit in before the v5 RC; if it's not possible, I'm fine with deferring it

Reply:
Yeah, makes sense to me.